File: Gready Algorithms/Solutions For 1 and 3/MoneyChange.java
package GreedyAlgorithms;
import java.util.Scanner;

public class MoneyChange {
    private static int getChange(int money) {
       int result=0;
       while (money>0){
           if (money>=10){
               money-=10;
           }else if (money>=5){
               money-=5;
           }else {
               money--;
           }
           result++;
       }
        return result;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int m = scanner.nextInt();
        System.out.println(getChange(m));

    }
}

File: Gready Algorithms/Solutions For 1 and 3/money_change.py
# python3


def money_change(money):
    result =0
    while money>0:
        if money>=10:
            money=money-10
        elif money>=5:
            money=money-5
        else:
            money=money-1
        result=result+1
    return result


if __name__ == '__main__':
    input_money = int(input())
    print(money_change(input_money))
